spring-boot/src/main/java/org/springframework/boot/context/embedded/Compression.java

```diff
@@ -31,12 +31,17 @@
 	private boolean enabled = false;
 
 	/**
-	 * Comma-separated list of MIME types that should be compressed.
+	 * MIME types that should be compressed.
 	 */
 	private String[] mimeTypes = new String[] { "text/html", "text/xml", "text/plain",
 			"text/css" };
 
 	/**
+	 * User agents for which responses should not be compressed
+	 */
+	private String[] excludedAgents = null;
```

Comment:
Similar to the comment above, this should read something like "Comma-separated list of user agents for which responses should not be compressed".